src/olympia/stats/views.py

```diff
@@ -134,7 +134,14 @@ def overview_series(request, addon, group, start, end, format, beta=False):
     check_stats_permission(request, addon, beta)
 
     dls = get_series(DownloadCount, addon=addon.id, date__range=date_range)
-    updates = get_series(UpdateCount, addon=addon.id, date__range=date_range)
+
+    if beta:
+        updates = get_updates_series(addon=addon,
+                                     start_date=date_range[0],
+                                     end_date=date_range[1])
+    else:
+        updates = get_series(UpdateCount, addon=addon.id,
```

Comment:
I decided not to change things in `get_series` because it is also used for download stats, and that's already messy enough.